andela/travel_tool_front
# #160238493 Fix details modal display on mobile

#### What does this PR do?
This PR fixes the request details modal that does not close on mobile view or display

#### Description of Task to be completed?
- Fix the z-index positioning in the SCSS files 
- Fix positioning of the modal header
- Fix text editor display on mobile.

#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- On the Requests page, click on the request Id from the list of requests displayed on the request table, you should be able to see the Request details Modal displayed.
- Right click on the page, choose inspect, and choose `Toggle mobile toolbar` to view the modal in mobile responsive views, the close button is displayed on the right-hand side of the modal to enable closing of the modal. 


#### Any background context you want to provide?
To test if this works, you will need to create a new request, therefore, ensure that both [backend](https://github.com/andela/travel_tool_back) API and frontend client are set up locally or you can test it on the staging environment after the PR is merged.

#### What are the relevant pivotal tracker stories?
[#160238493](https://www.pivotaltracker.com/story/show/160238493)


#### Screenshots (if appropriate)

<img width="493" alt="screen shot 2018-09-04 at 01 04 48" src="https://user-images.githubusercontent.com/12033494/45002696-5d71aa00-afe2-11e8-921e-783555f288c8.png">



#### Questions:
None
